types/span.go

```diff
@@ -46,3 +46,11 @@ func SortSpanByID(a []*Span) {
 		return a[i].ID < a[j].ID
 	})
 }
+
+// Valid is invoked to check validity of the span
+func (s *Span) Valid() (valid bool) {
+	if s.ID == 0 {
+		return
+	}
+	return true
+}
```

Comment:
Span with `ID=0` is also valid. That's the initial span embedded into genesis file.

Reply:
If I send id as zero, it panics. thats coz it fails genesis validation?